Bearded.UI/Controls/implementations/ListControl.cs

```diff
@@ -70,14 +70,21 @@ public IListItemSource ItemSource
             }
         }
 
+        public static ListControl ClickThrough(
```

Comment:
I'd also rename this to CreateClickThrough() or NewClickThrough(). Though I'm still wondering if doing this through the construction, instead of just having a property like with all other controls is the right move.

Reply:
I think this should be a static constructor method. The reason for this is that we are affecting the `IsClickThrough` property of two sub-controls here. Since they can be injected, there would suddenly be some strange side effects to setting this property on the list control. Also, if the two containers have different values for `IsClickThrough`, then what?

I think making this a shorthand constructor method is much clearer, especially in the context of injected containers.

Reply:
Good point. I agree that it should be possible to have the list view itself be click through, but maybe have one or both of the children not be.. or the other way around or... Not because I see a real use case for it, but because they are all controls and it is more consistent that way, and generally with how we decided to make everything behave the same as much as possible.